tensorflow/tools/test/performance.bzl

```diff
@@ -0,0 +1,55 @@
+# -*- Python -*-
+
+load("//tensorflow:tensorflow.bzl", "tf_py_test")
+
+# Create a benchmark test target of a TensorFlow C++ test (tf_cc_*_test)
+def tf_cc_logged_benchmark(
+    name=None,
+    target=None,
+    benchmarks="..",
+    tags=[],
+    test_log_output_prefix=""):
+  if not name:
+    fail("Must provide a name")
+  if not target:
+    fail("Must provide a target")
+  if (not ":" in target
+      or not target.startswith("//")
+      or target.endswith(":all")
+      or target.endswith(".")):
+    fail(" ".join(("Target must be a single well-defined test, e.g.,",
+                   "//path/to:test. Received: %s" % target)))
+
+  all_tags = list(set(tags) + set(["benchmark-test"]))
```

Comment:
Added "local" and "regression-test" tags as suggested. 

However, ci_paramerized_build.sh has its own test tag filtering mechanism, for other build configurations such as GPUs. It uses the --test_tag_filters flag of "bazel test". I will use --test_tag_filters=-benchmark-test to filter out the benchmark tests for non-benchmark jobs. Let me know it is okay.